docs/organizations/security/set-git-tfvc-repository-permissions.md

```diff
@@ -91,7 +91,7 @@ You can set the permissions for all Git repositories for a project, or for a sin
 
 ::: moniker range="vsts"  
 
-0. To set the set the permissions for all Git repositories for a project, choose **Git Repositories** and then choose the security group whose permissions you want to manage. 
+0. To set the set the permissions for TFVC repository for a project, choose **TFVC Repository** and then choose the security group whose permissions you want to manage. 
```

Comment:
Can you change this to say "... set the permissions for `a` TFVC repository" and then I will merge this.